### DISCUSSION TITLE
Is there a way to set the path for the JWT Refresh token cookie

### DISCUSSION THREAD

**Discussion by SmitharyZach** [micronaut-projects/micronaut-security#1445] [Q&A]
I am trying to access the JWT_REFRESH_TOKEN and the cookie has the path set to /oauth/access_token. I can update the endpoint path but would like to just update the cookie path. 

Token.JWT.Cookie.cooke-path is for the jwt token only. See the pic below:
<img width="676" alt="image" src="https://github.com/micronaut-projects/micronaut-security/assets/36892129/24b90c1f-79e0-4fe2-b4a1-092035cb23c1">

I am trying to access this cookie using Next JS server side rendering. When trying to refresh the token the request that comes from server side props is not /oauth/access_token so I can't get it to refresh it. Let me know if that makes sense or if you have any ideas. Thanks!

**Comment by sdelamo** [comment]
See guide [JWT refresh token ](https://guides.micronaut.io/latest/micronaut-security-jwt-gradle-java.html#issuing-a-refresh-token
